src/transaction/transaction.cpp

```diff
@@ -137,9 +149,10 @@ void Transaction::pushCatalogEntry(CatalogSet& catalogSet, CatalogEntry& catalog
                     catalogEntry.getType());
             }
         } break;
+        case CatalogEntryType::REL_GROUP_ENTRY:
         case CatalogEntryType::SEQUENCE_ENTRY: {
-            const auto sequenceCatalogEntry = catalogEntry.constPtrCast<SequenceCatalogEntry>();
-            wal->logDropCatalogEntryRecord(sequenceCatalogEntry->getOID(), catalogEntry.getType());
+            // TODO(Guodong): support alter rel group.
```

Comment:
Is this the right place for the TODO?

Reply:
Yes if not support ALTER. Then REL_GROUP and SEQUENCE share the same drop path.